server/database.js

```diff
@@ -0,0 +1,39 @@
+function dataEndpoint(pool, request, response) {
+  // const test_location = request.headers['x-teachermoments-location'];
+  // console.log(test_location)
```

Comment:
I left this here because I wasn't sure how to go about deciding what url to look for. For now, I hard code this function to use 
```
const location = `${domain}/teachermoments/turner?KevinTesting20180319`;
```

but in production, people should be able to query for any location. Is the idea to make a middleman function to first confirm that the current user has access to data with that url?

Reply:
I'm removing this code for now but I do think the location variable will end up coming from the header of the request

Reply:
Okay, so I commented over https://github.com/mit-teaching-systems-lab/threeflows/pull/348#discussion_r177618684, let's talk more over there :)

The main idea is that you can't trust the `location` param for the client blindly, and somewhere on the server has to check this (or just hardcode for now and build that second step in the next PR).

Reply:
got it!